sdk/storage/azure-storage-files-datalake/src/datalake_file_client.cpp

```diff
@@ -105,8 +105,8 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
   {
     auto parsedConnectionString = Azure::Storage::Details::ParseConnectionString(connectionString);
     auto fileUri = std::move(parsedConnectionString.DataLakeServiceUri);
-    fileUri.AppendPath(fileSystemName, true);
-    fileUri.AppendPath(filePath, true);
+    fileUri.AppendPath(fileSystemName);
```

Comment:
I think this should've been fixed earlier.

Reply:
Is this a breaking change?

Reply:
Strange, I commented this one before I resolved this.
No this is a bug fix just like previous releases.